### PR TITLE
Update login page logos

### PR DESCRIPTION
**Pull request by mhkc**
This PR updates the logos on the login page and make the swedac logo configurable. To alter the logo you have to put a new logo in scout/server/blueprints/public/static and change the variable i `ACCREDITATION_BADGE` in `config.py`

image

**To test**
1. Setup a scout demo instance
2. Verify that `ACCREDITATION_BADGE`
3. Verify that the swedac logo is being displayed on the login page
4. Set  `ACCREDITATION_BADGE` to null and restart the server
5. Verify that the logo is not displayed on the login page.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by mikaell** [comment]
Did you update the documentation?

**Comment by mhkc** [comment]
Nope, forgot to do that.  Ill fix it tomorrow

File: scout/server/blueprints/public/views.py [hunk 1]
```diff
@@ -21,7 +25,11 @@ def index():
     form = None
     if current_app.config.get("LDAP_HOST"):
         form = LDAPLoginForm()
-    return render_template("public/index.html", version=__version__, form=form)
+
+    badge_name = current_app.config.get("ACCREDITATION_BADGE")
```
Comment (mikaell) [inline scout/server/blueprints/public/views.py:29]: Check if badge image exists (here or elsewhere) and give reasonable warning, and either add placeholder text or hide the image element html.

**Review by mikaell — CHANGES_REQUESTED** [review]
Code Looks good and all tests pass -only needs to handle the case of badly configured image path (and maybe a pytest for that 🙂).

File: scout/server/blueprints/public/views.py [hunk 2]
```diff
@@ -21,7 +26,15 @@ def index():
...
+    if badge_name and not Path(public_bp.static_folder, badge_name).is_file():
```
Comment (mikaell) [inline scout/server/blueprints/public/views.py:31]: 👍

**Review by mikaell — APPROVED** [review]
Nice!

**Pull request merged**